### PR TITLE
fix(popover): fix issues withe scrolling with an open popover

### PR DESCRIPTION
**Pull request by flesher1**
Adds modifier to popover so that it will not flip while scrolling.

Closes AMLCS-76

### CODE REVIEW

**Comment by skhtet** [comment]
:+1: @mleeBoeing need another review

**Comment by mleeBoeing** [comment]
Two things that need to change
1. In Listing Management on the left hand side, click Contact Owners and the textbox will appear. The z-index needs to change.
2. In Bookmarks, click on the bookmark options and then scroll down.

File: src/components/bookmarks/BookmarkFolder.js [hunk 1]
```diff
@@ -225,12 +225,14 @@ class BookmarkFolder extends React.Component {
...
-                    <Icon style={{position: 'absolute', right: '50px', top: '25px', zIndex: '10'}} icon="people" title="Shared Folder" />
+                    <div className="tile-top">
+                        <Icon style={{position: 'absolute', right: '50px', top: '25px', zIndex: '10'}} icon="people" title="Shared Folder" className="tile-top"/>
+                    </div>
                 }
                 <a aria-label={"open " + title + " folder containing " + listings.length + " items"} href="javascript:;" onClick={this.toggleFolderDialog} />
-                <span style={{position: 'absolute', right: '20px', top: '20px', zIndex: '10'}}>
-                    <Popover content={menu} position={Position.BOTTOM_RIGHT} minimal={true}>
-                        <Button aria-label={"open " + title + " folder menu"} icon="more" title="Menu" text="" className="pt-button pt-minimal folder-options"/>
+                <span style={{position: 'absolute', right: '20px', top: '20px', zIndex: '9'}}>
```
Comment (mleeBoeing) [inline src/components/bookmarks/BookmarkFolder.js]: Though you didn't write them. Can you write the styling in the CSS file?

**Comment by mleeBoeing** [comment]
:+1:

**Pull request merged**